src/server/api/qrcode.ts

```diff
@@ -0,0 +1,48 @@
+import Express from 'express'
+import Joi from '@hapi/joi'
+import { createValidator } from 'express-joi-validation'
+
+import { ogUrl } from '../config'
+import createGoQrCode from '../util/qrcode'
+import ImageFormat from '../../shared/util/imageFormat'
+import { isValidShortUrl } from '../../shared/util/validation'
+
+function isValidFormat(format: string): boolean {
+  return (Object as any).values(ImageFormat).includes(format)
```

Comment:
sounds a bit hacky, was researching and saw [this link](https://stackoverflow.com/questions/42966362/how-to-use-object-values-with-typescript/42967397) which might be able to help circumvent this.

Reply:
I have changed it to this, which I think may be cleaner.
```
function isValidFormat(format: string): boolean {
  const validFormats = Object.values(ImageFormat) as string[]
  return validFormats.includes(format)
}
```